services/Skyra.Grpc/README.md

```diff
@@ -0,0 +1,168 @@
+<div align="center">
+
+![Skyra Logo](https://cdn.skyra.pw/gh-assets/skyra_avatar.png)
+
+# Skyra Project Grpc
+
+**GRPC Microservice to interact with Skyra Database**
+
+[![Discord](https://discord.com/api/guilds/254360814063058944/embed.png?style=banner2)](https://join.skyra.pw)
+
+</div>
+
+---
+
+**Table of Contents**
+
+-   [Skyra Project Grpc](#skyra-project-grpc)
+-   [Quick reference](#quick-reference)
+    -   [How to use this image](#how-to-use-this-image)
+        -   [Setting up a Docker Compose file](#setting-up-a-docker-compose-file)
+        -   [Using custom configuration](#using-custom-configuration)
+            -   [`POSTGRES_PASSWORD`](#postgres_password)
+            -   [`POSTGRES_USER`](#postgres_user)
+            -   [`POSTGRES_DB`](#postgres_db)
+            -   [`POSTGRES_HOST`](#postgres_host)
+            -   [`POSTGRES_PORT`](#postgres_port)
+    -   [Buy us some doughnuts](#buy-us-some-doughnuts)
+    -   [Contributors ✨](#contributors-%E2%9C%A8)
+
+---
+
+# Quick reference
+
+-   **Maintained by**:  
+    [Skyra Project](https://github.com/skyra-project)
+
+-   **Where to get help**:  
+    [the Skyra Lounge Discord server](https://join.skyra.pw/)
+
+-   **Where to file issues**:  
+    [https://github.com/skyra-project/skyra/issues](https://github.com/skyra-project/skyra/issues)
+
+-   **Source of this description**:  
+    [Skyra repo `services/Skyra.Grpc` directory](https://github.com/skyra-project/skyra/blob/main/services/Skyra.Grpc/README.md) ([history](https://github.com/skyra-project/skyra/commits/main/services/Skyra.Grpc/README.md))
+
+## How to use this image
+
+### Setting up a Docker Compose file
+
+```yaml
+version: '2.4'
+services:
+    grpc:
+        image: skyrabot/grpc:latest
+        container_name: 'skyra-grpc'
+        depends_on:
+            - postgres
+        environment:
+            - POSTGRES_HOST=postgres
+        ports:
+            - '8291:80'
+    postgres:
+        container_name: postgres
+        image: skyrabot/postgres:main
+        restart: always
+        ports:
+            - '5432:5432'
+        volumes:
+            - postgres-data:/var/lib/postgresql/data
+```
+
+[![Try in PWD](https://raw.githubusercontent.com/play-with-docker/stacks/master/assets/images/button.png)](https://raw.githubusercontent.com/skyra-project/skyra/feat/docker-dotnet/docker/playwithdocker-grpc-stack.yml)
+
+### Using custom configuration
+
+Both the GRPC and PostgreSQL images use several environment variables that you can configure
+
+#### `POSTGRES_PASSWORD`
+
+This optional environment variable is used for the `$POSTGRES_USER` to connect to the database. If it is not specified, then the default password of `postgres` will be used.
```

Comment:
Is the `$` in `POSTGRES_USER `intentional? Otherwise:

```suggestion
This optional environment variable is used for the `POSTGRES_USER` to connect to the database. If it is not specified, then the default password of `postgres` will be used.
```

Reply:
that's how you denote environment variables in bash, so yes.